### DISCUSSION TITLE
Missing test context in @fast-check/vitest

### DISCUSSION THREAD

**Discussion by SOULPWRD** [dubzzz/fast-check#5153] [Q&A] [answered]
Hi,

Vitest supports the [test context](https://vitest.dev/guide/test-context) value that is available as a parameter in the `it` or `test` functions. I've been using `@vitest/fast-check` lately and `it.prop` is missing this capability.

Maybe I'm missing something and the context value can be retrieved somehow. If not, would it be possible to implement this feature in the upcoming releases?

Thank you in advance.

**Comment by dubzzz** [comment, marked as answer]
The parameter is not forwarded to the caller at the moment. It might be added in the next iterations.

Just to let you know: we may drop the `it.prop` in favor of a dedicated `fuzz`. It would be more idiomatic with Vitest (same as the `bench`).
